spec/features/hub/edit_client_spec.rb

```diff
@@ -159,8 +159,8 @@
 
       within "#dependents-fields" do
         new_field_id = all(".dependent-form").last.first("input")["id"].tr('^0-9', '')
-        fill_in "hub_update_client_form_dependents_attributes_#{new_field_id}_last_name", with: "Chung"
-        select "2019", from: "hub_update_client_form_dependents_attributes_#{new_field_id}_birth_date_year"
+        fill_in "hub_update_client_form_dependents_attributes_#{new_field_id}_last_name", with: "Cranberry"
+        # fill_in "hub_update_client_form_dependents_attributes_#{new_field_id}_birth_date_year", with: "2019"
```

Comment:
Why commented-out?

Reply:
If the point is to test editing, let's edit this -- let's make it be 2018 and adjust the spec below.